### PR TITLE
Fixed bug #2370: Now showing correctly the payload of the request

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

**Comment by superalex** [comment]
Is this one ready to merge?

**Comment by jespino** [comment]
Sorry, yes, is ready

File: app/coffee/modules/admin/third-parties.coffee [hunk 1]
```diff
@@ -89,7 +89,7 @@ WebhookDirective = ($rs, $repo, $confirm, $loading) ->
                 for log in webhooklogs
                     log.validStatus = 200 <= log.status < 300
                     log.prettySentHeaders = _.map(_.pairs(log.request_headers), ([header, value]) -> "#{header}: #{value}").join("\n")
-                    log.prettySentData = JSON.stringify(log.request_data.data, undefined, 2)
+                    log.prettySentData = JSON.stringify(log.request_data)
                     log.prettyDate = moment(log.created).format("DD MMM YYYY [at] hh:mm:ss") # TODO: i18n
```
Comment (superalex) [inline app/coffee/modules/admin/third-parties.coffee]: Why did you remove the indentation parameters? (", undefined, 2"), it was nicer :)

Comment (jespino) [inline app/coffee/modules/admin/third-parties.coffee]: because the user can try to reproduce the hash with the payload, the the payload must be the same used to generate the hash

Comment (superalex) [inline app/coffee/modules/admin/third-parties.coffee]: You are right! :+1:
